acestep/core/generation/handler/prompt_utils.py

```
@@ -0,0 +1,162 @@
+        actual_language = vocal_language
+
+        if metas is not None:
+            try:
+                if isinstance(metas, str):
+                    parsed_metas = self._parse_metas([metas])
+                    meta_dict = parsed_metas[0] if parsed_metas and isinstance(parsed_metas[0], dict) else {}
+                elif isinstance(metas, dict):
+                    meta_dict = metas
+                else:
+                    meta_dict = {}
+            except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)